### PR TITLE
EP6-3297 My Sites Active Tab

### PR DESCRIPTION
**Pull request by zbroadhead**

### CODE REVIEW

**Comment by zbroadhead** [comment]
While fixing the defect with these styles, I was able to clean them up/simplify a bit too hence do many line differences.

**Comment by zbroadhead** [comment]
@ponddesign mdx entry added!

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "react-bootstrap": "^0.32.4",
     "react-content-loader": "^4.2.1",
     "react-dom": "^16.8.0",
+    "react-router-dom": "^5.0.1",
```
Comment (ponddesign) [inline package.json]: We shouldn't need `react-router-dom` for this. Let's uninstall this back out.

File: src/Components/Navs.mdx [hunk 2]
```diff
@@ -83,3 +84,40 @@ They can be `justified` to take the full width of their parent.
     </NavDropdown>
   </Nav>
 </Playground>
+
+## Nav underline style ![Not Ready Badge](https://img.shields.io/badge/component-not%20ready-red.svg#badge)
+
+Underline styled tabs can be build with Router, li, and Navlink elements by using "nav nav-tabs-underline" on a parent and an li element containing a NavLink or Link element.
+
+<Playground>
+  <Router>
```
Comment (ponddesign) [inline src/Components/Navs.mdx]: We don't need Router / div / ul here. Instead it is preferred to follow the React-Bootstrap API:

```
<Nav className="nav-slider-wrapper">
  <NavItem active>Tab 1</NavItem>
  <NavItem>Tab 2</NavItem>
  <NavItem>Tab 3</NavItem>
</Nav>
```

https://5c507d49471426000887a6a7--react-bootstrap.netlify.com/components/navs/#navs-dropdown

File: package.json (hunk 1 above)
Comment (zbroadhead) [reply package.json]: Put it in do be able to use the styles written for that in the mdx file, but I can probably do it differently.

File: src/Components/Navs.mdx (hunk 2 above)
Comment (zbroadhead) [reply src/Components/Navs.mdx]: See above, but I'll give it a shot.

Comment (zbroadhead) [reply src/Components/Navs.mdx]: done

File: src/Components/Navs.mdx [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 menu: Components
 ---
-
+import { useState } from 'react';
 import Nav from 'react-bootstrap/lib/Nav';
 import NavItem from 'react-bootstrap/lib/NavItem';
 import NavDropdown from 'react-bootstrap/lib/NavDropdown';
 import MenuItem from 'react-bootstrap/lib/MenuItem';
 import { Playground } from 'docz';
+import classnames from 'classnames';
```
Comment (ponddesign) [inline src/Components/Navs.mdx]: - [ ] Lib `classnames` imported, but not used, let's remove this line.

File: src/Components/Navs.mdx [hunk 4]
```diff
@@ -83,3 +85,28 @@ They can be `justified` to take the full width of their parent.
...
+              activeKey={activeKey}
+              onSelect={e => setActiveKey(e)}
+            >
+              <NavItem eventKey="1">All</NavItem>
+              <NavItem eventKey="2">eCommerce</NavItem>
+              <NavItem eventKey="3">Blog</NavItem>
+              <NavItem eventKey="4">Portfolio</NavItem>
+              <NavItem eventKey="5">Entertainment</NavItem>
+              <NavItem eventKey="6">Food &amp; Drink</NavItem>
+              <NavItem eventKey="7">News</NavItem>
```
Comment (ponddesign) [inline src/Components/Navs.mdx]: - [ ] One example should demonstrate the `disabled` state.

**Pull request merged**